definitions/vertical_scaling.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Vertical Scaling
+status: Completed
+category: concept
+---
+## Canary Deployment
+
+### What it is
+Vertical scaling refers to adding more resources (CPU/RAM/DISK) to your server (database or application server is still remains one) as on demand.
```

Comment:
Need to change this line to make it more general. Ideally replace everything after your to "application instance".